torch/_dynamo/variables/flash_attention.py

```diff
@@ -0,0 +1,26 @@
+from ..bytecode_transformation import create_call_function
+from .base import VariableTracker
+
+
+class SDPAParamsVariable(VariableTracker):
```

Comment:
Hmm, I don't love things like dynamo knowing about stuff like flash attention.

What do we think about giving users the ability to add their own type mapping? Does that leak too many VT internal notions? @ezyang ?

Reply:
Also nit about the naming but this doesn't really have to do with flash_attention but really about the general op "scaled_dot_product_attention"

Reply:
I'm fine with renaming as appropriate. To Voz's comment I'm not sure there is a way that would be easily understandable to users. We could limit this to types that have some constraints on the initialization type signature, but that seems like there will be more edge cases and we'd have to explain that to users anyway. Open to ideas here though.